PhysicsTools/SelectorUtils/interface/VersionedIdProducer.h

```diff
@@ -102,16 +105,21 @@ VersionedIdProducer(const edm::ParameterSet& iConfig) {
     edm::LogWarning("IdInformation")
       << idmsg.str();
 
+    
+
     produces<std::string>(idname);
     produces<edm::ValueMap<bool> >(idname);
     produces<edm::ValueMap<float> >(idname); // for PAT
-    produces<edm::ValueMap<unsigned> >(idname);
+    produces<edm::ValueMap<unsigned> >(idname);  
+    produces<edm::ValueMap<unsigned> >(idname+std::string(bitmap_label));
```

Comment:
Why not just use what you did already "Bitmap" Is there any concern this gets confused?